src/pages/Employees/components/EmployeeModal/EmployeeModal.tsx

```diff
@@ -238,7 +238,7 @@ export default function EmployeeModal({
             </HStack>
 
             <FormControl>
-              <FormLabel>Skills</FormLabel>
+              <FormLabel htmlFor="">Skills</FormLabel>
```

Comment:
Without the htmlFor explicitly stated here, the label was being applied to the first checkbox in the skills list, which causes a test failure in the edit modal.